benchmark/launch_software.sh

```diff
@@ -224,3 +245,60 @@ then
 else
     echo "${cmd}"
 fi
+
+## run DarkHorse
+cmd="${init_command}; \
+      bash ${scripts_dir}/run_darkhorse.sh ${nr_fp} \
+                                           ${diamond_db_nr} \
+                                           ${diamond_tabular_query} \
+                                           ${darkhorse_config_fp} \
+                                           ${darkhorse_install_dp} \
+                                           ${query_species_coding_seqs_fp} \
+                                           ${working_dir} \
+                                           ${threads}"
+if [ "${qsub_env}" == "true" ]
```

Comment:
This block is repeated 3 times, perhaps it will be worth making a function? Note that the only difference is the name of the job but you could get that from cmd, right?